playbooks/maas-agent-setup.yml

```diff
@@ -41,9 +42,9 @@
       file:
         path: "{{ maas_plugin_dir }}"
         state: directory
-        owner: "root"
+        owner: "{{ ansible_ssh_user | default('root')}}"
         group: "root"
-        mode: "0755"
+        mode: "0757"
```

Comment:
Is there a reason this was changed from `0755` to `0757`? We shouldn't want these writeable by everyone.

Reply:
This is what's there left in the feature branch:
https://github.com/rcbops/rpc-maas/pull/548/files#diff-91e2993f1af4f36c72349d76a45c789dR152

I vaguely recall it's that if we don't use 757,  we will run into a permission issue ?

Reply:
Changed back to 755